docs/odh_and_opf.md

```diff
@@ -0,0 +1,12 @@
+# What is Open Data Hub?
+
+[Open Data Hub](http://opendatahub.io/) (ODH) is an open source project based on Kubeflow that provides open source AI tools for running large and distributed AI workloads on OpenShift Container Platform. Currently, the Open Data Hub project provides open source tools for data storage like Ceph S3, distributed AI and Machine Learning (ML) workflows such as Kubeflow, Jupyter Notebook development environment and monitoring for data science users.
+
+
+# What is Operate First?
+*“A concept to incorporate operational experience into software projects.”*
+
+[Operate First](https://www.operate-first.cloud/) is an initiative to “operate” software in a production-grade environment - bringing users, developers and operators closer together. It is a concept where open source developers bring their projects to a production cloud during development where they can interact with a cloud provider’s operators and gain valuable feedback on the operational considerations of their project.  It uses the same community-building process of open-source projects, but extended to ops procedures and data. 
+
+Operate First manages a ready-to-use deployment of Open Data Hub on the [Mass Open Cloud](https://massopen.cloud/)(MOC) Smaug cluster which is available for the public to experience. [Here](https://www.operate-first.cloud/apps/content/odh/README.html) is a list of components that are currently available for anybody to start using.
```

Comment:
Take out reference to `Smaug` as the specific cluster name will change in the future.

Reply:
Done :)